src/AltinnCore/Common/Services/Implementation/CompilationSI.cs

```diff
@@ -47,7 +47,7 @@ public class CompilationSI : Interfaces.ICompilation
         /// <param name="configuration">The configuration.</param>
         /// <param name="partManager">The part manager.</param>
         /// <param name="compilationService">The compilation service.</param>
-        /// <param name="repositoryService">The service repository service.</param>
+        /// <param name="repositoryService">The app repository service.</param>
```

Comment:
No it is not the app's repository service. It is the repository service to talk with the repository.